tests/ga/test_exthandlers.py

```diff
@@ -9,7 +9,7 @@
 from azurelinuxagent.common.exception import ProtocolError, ExtensionError, ExtensionErrorCodes
 from azurelinuxagent.common.event import WALAEventOperation
 from azurelinuxagent.common.utils.processutil import TELEMETRY_MESSAGE_MAX_LEN, format_stdout_stderr
-from azurelinuxagent.common.cgroups import CGroups
```

Comment:
there are many changes in exthandlers.py that don't seem to have a corresponding unit test here